apps/vr-tests-react-components/src/stories/Field.stories.tsx

```diff
@@ -3,6 +3,7 @@ import { Steps, StoryWright } from 'storywright';
 import { storiesOf } from '@storybook/react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 7 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26488&commitId=48471804cba82ab823294dc907622517b76f9494&env=prod)

✅ There was 7 screenshots added, 0 screenshots removed, 1775 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DropdownField converged.base.default.chromium.png|0| Added|
|DropdownField converged.hint.default.chromium.png|0| Added|
|DropdownField converged.horizontal.default.chromium.png|0| Added|
|DropdownField converged.required.default.chromium.png|0| Added|
|DropdownField converged.size-large.default.chromium.png|0| Added|
|DropdownField converged.size-small.default.chromium.png|0| Added|
|DropdownField converged.validation.default.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>